.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
if we decide to also add .vscode settings, then we should document it in dev-rel-docs

Reply:
I created an issue for this: https://github.com/ai-cfia/dev-rel-docs/issues/40